src/modules/agora/AgoraClient.ts

```diff
@@ -0,0 +1,125 @@
+/*******************************************************************************
+
+    The class that recover data
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import axios, { AxiosInstance, AxiosResponse, AxiosError } from 'axios';
+import URI from "urijs";
+
+/**
+ * The class that recover data
+ */
+export class AgoraClient
+{
+    /**
+     * The instance of the axios
+     */
+    private client: AxiosInstance;
+
+    /**
+     * The URL encoder
+     */
+    private uri: URI;
+
+    /**
+     * Constructor
+     * @param host - The network address to connect to the Agora
```

Comment:
You can either write "connect to Agora" or "connect to the Agora node", but not "connect to the Agora".
Likewise, you can say "going to Samsung" or "going to the Samsung headquarter", not "going to the Samsung".

Reply:
I changed it